string dest_file_name
---
bool return_value